windows/flutter/generated_plugin_registrant.cc

```
@@ -6,6 +6,12 @@
 
 #include "generated_plugin_registrant.h"
 
+#include <cloud_firestore/cloud_firestore_plugin_c_api.h>
+#include <firebase_core/firebase_core_plugin_c_api.h>
 
 void RegisterPlugins(flutter::PluginRegistry* registry) {
```

cppcheck

[style] The function 'RegisterPlugins' is never used.

(unusedFunction)